### DISCUSSION TITLE
Question: Is there a way to use BootstrapVue with VTU2?

### DISCUSSION THREAD

**Discussion by fgsalomon** [vuejs/test-utils#2064] [General]
Hi,

I've some projects that use BootstrapVue that I'd like to migrate to Vue 3. Using `@vue/compat` I can run these projects without issues but I couldn't find a way to migrate their tests to the new tools.

Initially I was using  Vue Testing Library v5 (VTU1)

**Describe the question**

Is it possible to use VTU2 with Vue 2 plugins like BootstrapVue and `@vue/compat`?

**To Reproduce**
I've set a minimal repo here: https://github.com/fgsalomon/vue3-bootstrap-vue-issues

This repo contains a component [TestBootstrap.vue](https://github.com/fgsalomon/vue3-bootstrap-vue-issues/blob/main/src/components/TestBootstrap.vue) that uses BootstrapVue components. 
The component has a unit [test](https://github.com/fgsalomon/vue3-bootstrap-vue-issues/blob/main/src/components/__tests__/TestBootstrap.spec.ts) with Vue Testing Library v7 (VTU2).

**Expected behavior**
The test runs successfully.

Currently `yarn test:unit` fails with the error when trying to install BootstrapVue plugin: 
```
TypeError: Vue.extend is not a function
```

**Related information:**
```
  System:
    OS: macOS 13.3.1
    CPU: (8) arm64 Apple M1 Pro
    Memory: 95.05 MB / 16.00 GB
    Shell: 5.9 - /bin/zsh
  npmPackages:
    @testing-library/vue: ^7.0.0 => 7.0.0 
    @vue/test-utils: ^2.3.2 => 2.3.2 
    vitest: ^0.31.0 => 0.31.0 
    vue: ^3.3.2 => 3.3.2 
``` 

Thank you.

**Comment by cexbrayat** [comment]
I suppose it should work, as @xanf wrote the compact mode for VTU and is one of the authors of BootstrapVue
But I have to admit I don't use it myself, so I can't really help.

So let's friendly ping @xanf here to have his opinion, bu you'll probably have more luck opening thiss issue on BootstrapVue directly.
